modules/websocket/emws_server.h

```diff
@@ -42,6 +42,7 @@ class EMWSServer : public WebSocketServer {
 
 public:
 	Error set_buffers(int p_in_buffer, int p_in_packets, int p_out_buffer, int p_out_packets) override;
+	void set_extra_headers(const Vector<String> &p_headers);
```

Comment:
This needs to be marked as override
```suggestion
	void set_extra_headers(const Vector<String> &p_headers) override;
```